float64 p1
float64 p2
float64 p3
float64 time
string interpolacja
---
string wynik
